### ISSUE TITLE
[APP SUBMITTED]: cannot find 'cur_ep_enddate'

### ISSUE THREAD

**Issue by ubun2Junky** [SiCKRAGETV/sickrage-issues#1402]
### INFO
Python Version: **2.7.6 (default, Mar 22 2014, 22:59:56) [GCC 4.8.2]**
Operating System: **Linux-3.13.0-49-generic-x86_64-with-Ubuntu-14.04-trusty**
Locale: unknown
Branch: **master**
Commit: SiCKRAGETV/SickRage@281409d366b228a455879e02b2d55d3967db29e4
Link to Log: https://gist.github.com/613438fdca465191a299
### ERROR
```
Thread-58 :: Failed doing webui callback: Traceback (most recent call last):
  File "/opt/sickrage/sickbeard/webserve.py", line 258, in async_call
    result = function(**kwargs)
  File "/opt/sickrage/sickbeard/webserve.py", line 539, in comingEpisodes
    return t.respond()
  File "/opt/sickrage/cache/cheetah/_opt_sickrage_gui_slick_interfaces_default_comingEpisodes_tmpl.py", line 748, in respond
    if VFFSL(SL,"cur_ep_enddate",True) < VFFSL(SL,"today",True): # generated from line 377, col 13
NotFound: cannot find 'cur_ep_enddate'

```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by fernandog** [comment]
@ubun2Junky did you update using git pull??

**Comment by ubun2Junky** [comment]
I stopped sickrage services, run 'git pull'.  It did download the newer files.  Once it was completed, I restarted the services.  Sickrage started like normal, but when I tried to view 'coming episodes', I continue to get the above mentioned error.

**Comment by fernandog** [comment]
@ubun2Junky stop SR, delete Sessions folder inside SickRage/cache

**Comment by ubun2Junky** [comment]
Stopped services and removed the Sessions folder inside sickrage/cache
directory.  Restarted services, tried 'Coming Episodes', same error as
before.

L

**Comment by fernandog** [comment]
@ubun2Junky read my comment again. And never git pull.

**Comment by ubun2Junky** [comment]
I really appreciate your patience in helping me out with this.  I did read your comment, I just missed typing the cheetah directory, sry.  I normally leave sickrage alone and don't update via 'git pull', I just noticed the issue when I try to pull 'Coming Episodes', so I thought I'd try to update my version of SR.  I have it to check for updates every 12hrs, but wasn't sure it was picking up the updates.  I went into the advanced settings and Branch version had a red comment "Error: No branches found".  I've also made sure that the git reset is enabled.  I ran into some instructions you posted on how to solve update problems and I ran the commands.  Still nothing changed so I thought I'd seek help here.  Perhaps I just need to reinstall the app?
